docs/mediapipe.md

```diff
@@ -0,0 +1,141 @@
+
+For more details you can visit mediapipe concept description - [Calculators Concept Page](https://developers.google.com/mediapipe/framework/framework_concepts/calculators) or Ovms specific calculators implementation - [Ovms Calculators Concept Page](https://github.com/openvinotoolkit/model_server/blob/releases/2023/0/src/mediapipe_calculators/calculators.md)
+
+## Graph proto files <a name="graph-proto"></a>
+
+Graph proto files are used to define a graph. Example content of graph containing OVMSCalculator node proto file:
+
+```
+
+input_stream: "in"
+output_stream: "out"
+node {
+  calculator: "OVMSOVCalculator"
+  input_stream: "B:in"
+  output_stream: "A:out"
+  node_options: {
+        [type.googleapis.com / mediapipe.OVMSCalculatorOptions]: {
+          servable_name: "dummy"
+          servable_version: "1"
+          tag_to_input_tensor_names {
+            key: "B"
+            value: "b"
+          }
+          tag_to_output_tensor_names {
+            key: "A"
+            value: "a"
+          }
+        }
+  }
+}
+
+
+Here can be found more information about [MediaPipe graphs proto](https://developers.google.com/mediapipe/framework/framework_concepts/graphs)
+
+```
+
+## Configuration files <a name="configuration-files"></a>
+MediaPipe graph configuration is to be placed in the same json file like the 
+[models config file](starting_server.md).
+While models are defined in section `model_config_list`, graphs are configured in
+the `mediapipe_config_list` section. 
+Nodes in the MediaPipe graphs can reference both to the models configured in model_config_list section and in subconfig.
+
+Basic graph section template is depicted below:
+
+```
+
+{
+    "model_config_list": [...],
+    "mediapipe_config_list": [
+    {
+        "name":"mediaDummy",
+        "base_path":"/mediapipe/graphs/",
+        "graph_path":"graphdummyadapterfull.pbtxt",
+        "subconfig":"subconfig_dummy.json"
+    }
+    ]
+}
+
+```
+
+Basic subconfig:
+
+```
+
+{
+    "model_config_list": [
+        {"config": {
+                "name": "dummy",
+                "base_path": "/models/dummy",
+                "shape": "(1, 10)"
+        	}
+	}
+    ]
+}
+
+
+```
+
+
+### MediaPipe configuration options explained
+
+|Option|Type|Description|Required|
+|:---|:---|:---|:---|
+|`"name"`|string|Graph identifier related to name field specified in gRPC/REST request|Yes|
+|`"base_path"`|string|Path to the which graph definition and subconfig files paths are relative. May be absolute or relative to the main config path. Default value is "{main config path}\"|No|
+|`"graph_path"`|string|Path to the graph proto file. May be absolute or relative to the base_path. Default value is "{base_path}\graph.pbtxt". File have to exist.|No|
+|`"subconfig"`|string|Path to the subconfig file. May be absolute or relative to the base_path. Default value is "{base_path}\subconfig.json". Non existence of file does not result in error.|No|
```

Comment:
```suggestion
|`"subconfig"`|string|Path to the subconfig file. May be absolute or relative to the base_path. Default value is "{base_path}\subconfig.json". Missing file does not result in error.|No|
```
This does not produce error only if this field is optional and missing or also when the subconfig field is specified?
In 2nd case I think we should actually report error (this can be done in production release).